include/seqan3/core/detail/strong_type.hpp

```diff
@@ -87,10 +87,10 @@ namespace seqan3::detail
  * A strong type is expressive in what it actually represents as a value.
  * In our toy example we could define two strong types as follows:
  *
- * \snippet test/snippet/core/detail/strong_type_2.cpp error_window
+ * \include test/snippet/core/detail/strong_type_2_error_window.cpp
```

Comment:
Please omit the _2 in the filename (and rename the file, of course). Now they have unambiguous names, so it's not needed anymore.